ReactiveCocoa/CocoaTarget.swift

```diff
@@ -4,16 +4,56 @@ import enum Result.NoError
 
 /// A target that accepts action messages.
 internal final class CocoaTarget<Value>: NSObject {
+	private enum State {
+		case idle
+		case sending
+		case replayAfterSending([Value])
+	}
```

Comment:
I think it would probably simplify the code do this:

```swift
private enum State {
    case idle
    case sending(queue: [Value])
}
```

What do you think?